lib/solar_system.rb

```diff
@@ -0,0 +1,74 @@
+require_relative "planet"
+
+class SolarSystem
+  attr_reader :star_name, :planets
+
+  def initialize(star_name)
+    @star_name = star_name
+    @planets = []
+    @planet_print = ["Planets orbiting #{star_name}"]
+    #@new_planet
+  end
+
+  def add_planet(planet)
+    @planets << planet
+  end
+
+  def add_new_planet
+    print "Okay, what's the name of the planet we're missing?\n"
```

Comment:
This method should be moved to `main.rb` That way you would separate the user interface role with the job of keeping and tracking the list of planets.